CONTRIBUTING.md

```diff
@@ -86,6 +86,16 @@ Some tips for hosting a great review club meeting:
 - Have fun, and pat yourself on the back for making Bitcoin protocol development
   stronger and more decentralized 🚀
 
+### After the Meeting
+
+One of the review club maintainers will:
+
+- Copy the meeting log to the `##Meeting log` section of the PR page. Meeting
+  logs should be copied exactly, but an `#Erratum` section can be added to
+  correct bad factual errors.
+
+- Change the `status` of the PR page to `past`.
```

Comment:
Suggestions:
- `## Meeting Log`
- `## Erratum`
- lines 93 and 97, perhaps replace "PR page" with "post" or "meeting post"

Reply:
fixed!